pkg/validator/server_benchmark_test.go

```diff
@@ -0,0 +1,46 @@
+package validator
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/openservicemesh/osm/pkg/logger"
+	"github.com/openservicemesh/osm/pkg/webhook"
+)
+
+func BenchmarkDoValidation(b *testing.B) {
+	b.StopTimer()
+	if err := logger.SetLogLevel("error"); err != nil {
+		b.Logf("Failed to set log level to error: %s", err)
+	}
+
+	req := &http.Request{
+		Header: map[string][]string{
+			webhook.HTTPHeaderContentType: {webhook.ContentTypeJSON},
+		},
+		Body: io.NopCloser(strings.NewReader(`{
+		"metadata": {
+			"uid": "some-uid"
+		},
+		"request": {}
+	}`)),
+	}
+
+	w := httptest.NewRecorder()
+	s := &validatingWebhookServer{
+		validators: map[string]validateFunc{},
```

Comment:
Looks like there's no validations here meaning no actual work will be done. Let's instantiate the server the same way we normally do

Reply:
Validations added.